### ISSUE TITLE
wrong code at t2e/lstm_classifier.py

### ISSUE THREAD

**Issue by soki927** [Demfier/multimodal-speech-emotion-recognition#38]
42 # sum hidden states
43    class_scores = F.softmax(self.out(rnn_output), dim=1)

↓you should change

42 # sum hidden states
43    class_scores = F.log_softmax(self.out(rnn_output), dim=1)

**Comment by raniaahmed123** [comment]
please help for this missed file :
pred_probas/text_lstm_classifier.pkl
